### PR TITLE
Added Universal Code Search blog post for GitLab

### PR DESCRIPTION
**Pull request by ryan-blunden**
Blog post with embedded video for configuring GitLab as a code host.

### CODE REVIEW

File: blogposts/universal-code-search-gitlab.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  blog
+]
+slug: universal-code-search-gitlab
+heroImage: /external-logos/gitlab-mark.svg
+published: true
+---
+
+Sourcegraph [Universal Code Search](/universal-code-search) enables developers to explore and better understand all code, faster, with contextual code intelligence.
+
+This screencast (with accompanying audio) shows how to [connect and configure](https://docs.sourcegraph.com/admin/external_service/gitlab) Sourcegraph Universal Code Search for public and private repositories on GitLab.com and GitLab CC/EE instances
```
Comment (christinaforney) [inline blogposts/universal-code-search-gitlab.md]:
```suggestion
This screencast (with accompanying audio) shows how to [connect and configure](https://docs.sourcegraph.com/admin/external_service/gitlab) Sourcegraph Universal Code Search for public and private repositories on GitLab.com and GitLab CE/EE instances
```

Comment (christinaforney) [reply blogposts/universal-code-search-gitlab.md]: Perhaps this is more clear in the screencast, but there's a big difference between enabling for my account on GitLab.com and all the other configuration/admin steps when running your own GitLab instance. Can you make sure to distinguish between the two here?

File: blogposts/universal-code-search-gitlab.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+In addition to connection configuration, it discusses important factors for determining which repositories Sourcegraph will index, such as excluding archived repositories, as well as repositories based on naming patterns such `deploy-`.
```
Comment (christinaforney) [inline blogposts/universal-code-search-gitlab.md]:
```suggestion
In addition to configuring the Sourcegraph-GitLab connection, the screencast discusses important factors for setting up your Sourcegraph instance, such as determining which repositories Sourcegraph will index, excluding archived repositories, or including repositories based on naming patterns (e.g.,`deploy-`).
```

File: blogposts/universal-code-search-gitlab.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/397320487?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://www.youtube.com/watch?v=r8kvCUeeMKQ" target="_blank">Watch on YouTube</a></p>
+</p>
+
+See the [Sourcegraph GitLab documentation](https://docs.sourcegraph.com/admin/external_service/gitlab) for full configuration options, including [enabling repository permissions](https://docs.sourcegraph.com/admin/external_service/gitlab#repository-permissions).
+
+The script used in the screencast to search for repositories to ignore is available as a [GitLab snippet](https://gitlab.com/snippets/1952534).
```
Comment (christinaforney) [inline blogposts/universal-code-search-gitlab.md]: This reads awkwardly but I don't have an improvement. There are some typos in the comment of your script, and I think there's a stray 'f' on line 14.

Comment (vanesa) [inline blogposts/universal-code-search-gitlab.md]:
```suggestion
You can use this [Python script](https://gitlab.com/snippets/1952534) used in the screencast to search for repositories you want to ignore.
```

File: blogposts/universal-code-search-gitlab.md (hunk 1 above)
Comment (ryan-blunden) [reply blogposts/universal-code-search-gitlab.md]: Good call re GitLab.com vs. GitLab self-hosted. I'll change the wording so it's specific to CE/EE.

File: blogposts/universal-code-search-gitlab.md (hunk 3 above)
Comment (ryan-blunden) [reply blogposts/universal-code-search-gitlab.md]: Thanks, fixed the typos in the script and I'll re-work this sentence.

File: blogposts/universal-code-search-gitlab.md (hunk 2 above)
Comment (ryan-blunden) [reply blogposts/universal-code-search-gitlab.md]: I incorporated most of these changes, thanks!

File: blogposts/universal-code-search-gitlab.md (hunk 3 above)
Comment (ryan-blunden) [reply blogposts/universal-code-search-gitlab.md]: I took your idea and incorporated it into a new version of this sentence. Thanks!

**Pull request merged**